zerver/views/development/registration.py

```diff
@@ -14,9 +14,10 @@
 from zerver.views.registration import accounts_register
 
 
-# This is used only by the Puppeteer test in 01-realm-creation.js.
+# This is used only by the Puppeteer test in realm-creation.ts.
 def confirmation_key(request: HttpRequest) -> HttpResponse:
-    return json_success(request, data=request.session.get("confirmation_key"))
+    assert "confirmation_key" in request.session
+    return json_success(request, data=request.session["confirmation_key"])
```

Comment:
`session[…]` already raises `KeyError` on absence, so the separate `assert` doesn’t add anything.